daemon/info.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/docker/docker/utils"
 )
 
-func (daemon *Daemon) CmdInfo(job *engine.Job) error {
+func (daemon *Daemon) Info() (*engine.Env, error) {
```

Comment:
Is there any reason we're not typing all of the results here so its not just a list of strings being returned to the client?